Shell/AnswerExtractor.cpp

```diff
@@ -36,6 +38,9 @@
 
 #include "AnswerExtractor.hpp"
 
+namespace {
```

Comment:
...uh. C++ name resolution makes my head hurt, can you explain this anonymous namespace for me?

Reply:
As discussed in person, I removed it.